### ISSUE TITLE
[Error] Please specify the running stage

### ISSUE THREAD

**Issue by MonroeHe** [open-mmlab/Amphion#58]
when I running the commend(following as the guide: https://huggingface.co/amphion/valle_libritts )
'''
sh egs/tts/VALLE/run.sh --stage 3 --gpu "0" \
    --config "ckpts/tts/valle_libritts/args.json" \
    --infer_expt_dir Amphion/ckpts/tts/valle_libritts \
    --infer_output_dir Amphion/ckpts/tts/valle_libritts/result \
    --infer_mode "single" \
    --infer_text "This is a clip of generated speech with the given text from Amphion Vall-E model." \
    --infer_text_prompt "But even the unsuccessful dramatist has his moments." 
'''
I got the following error message:
[Error] Please specify the running stage

**Comment by lmxue** [comment]
Sorry, but I can't reproduce the error you mentioned.  Could you please provide more details or error messages?

**Comment by MonroeHe** [comment]
run script on mac(m2) and shell is zsh

❯ sh egs/tts/VALLE/run.sh --stage 3 --gpu "0" \
--config "ckpts/tts/valle_libritts/args.json" \
--infer_expt_dir Amphion/ckpts/tts/valle_libritts \
--infer_output_dir Amphion/ckpts/tts/valle_libritts/result \
--infer_mode "single" \
--infer_text "This is a clip of generated speech with the given text from Amphion Vall-E model." \
--infer_text_prompt "many animals of even complex structure which live parasitically within others are wholly devoid of an alimentary cavity" \
--infer_audio_prompt ckpts/tts/valle_libritts/prompt/LJ025-0076.wav
[Error] Please specify the running stage
❯

**Comment by butzhang** [comment]
I encounter this problem before, 
the problem is that we need to install 
```
brew install gnu-getopt
```
on mac 
and then source it.

**Comment by ostinsolo** [comment]
Hi, I encountered same issues here, fortunately I managed to solve it! But I have a question regarding filetype. I should find in the directory prompt_example 
Amphion/egs/tts/VALLE/prompt_examples'
(amphion-env) .../.../ ls
260_123440_000010_000004.normalized.txt 6829_68771_000027_000000.normalized.txt
5142_33396_000002_000004.normalized.txt 7176_92135_000004_000000.normalized.txt

Those files should be .wav but I have .txt files. What type of audio files should I input and why am I facing this issue?
